# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pam/Linux-PAM-1.1.0-lastlog-fix.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://bugs.gentoo.org/attachment.cgi?id=195319

--- Linux-PAM-1.1.0/modules/pam_lastlog/pam_lastlog.c.orig
+++ Linux-PAM-1.1.0/modules/pam_lastlog/pam_lastlog.c
@@ -471,7 +471,7 @@ last_login_failed(pam_handle_t *pamh, in
 		failed),
 	    failed);
 #else
-	if (daysleft == 1)
+	if (failed == 1)
 	    retval = asprintf(&line,
 		_("There was %d failed login attempt since the last successful login."),
 		failed);
